app/src/test/java/org/simple/clinic/newentry/PatientEntryUpdateTest.kt

```diff
@@ -316,4 +316,30 @@ class PatientEntryUpdateTest {
         )
   }
 
+  @Test
+  fun `when the age exceeds max limit, then show error`() {
+    //given
+    val errors: List<PatientEntryValidationError> = listOf(PatientEntryValidationError.AGE_EXCEEDS_MAX_LIMIT)
+    val givenModel = defaultModel
+        .fullNameChanged("Name")
+        .ageChanged("130")
```

Comment:
This can cause problems if we change the max allowed age in the app to more than 130. The test will have to be updated because of a hidden dependency which is not exposed in this test. 

We have declared a global constant for the max age (`MAX_ALLOWED_PATIENT_AGE`). Could you please use this value instead?

```kotlin
.ageChanged(MAX_ALLOWED_PATIENT_AGE.plus(1).toString())
```

Reply:
resolved

Reply:
Resolved